package.nls.json

```diff
@@ -21,6 +21,25 @@
     "AWS.configuration.description.samcli.debug.attach.timeout": "Maximum amount of time to wait for debugger to connect to a SAM Local debug run (in milliseconds)",
     "AWS.configuration.description.samcli.location": "Location of SAM CLI. SAM CLI is used to create, build, package, and deploy Serverless Applications. [Learn More](https://aws.amazon.com/serverless/sam/)",
     "AWS.configuration.description.telemetry": "Enable the AWS Toolkit to send usage data to AWS. `\"Use IDE settings\"` respects the value specified in \"`telemetry.enableTelemetry`\"",
+    "AWS.configuration.description.awssam.debug.target": "The type of invocation to launch. Possible values:\n* `template` uses a SAM Template as an entrypoint\n* `code` invokes Lambda code directly.",
+    "AWS.configuration.description.awssam.debug.lambdaHandler": "Lambda Function handler to invoke.",
+    "AWS.configuration.description.awssam.debug.projectRoot": "The root of the project, used to determine where in the file-system to locate the lambdaHandler.",
+    "AWS.configuration.description.awssam.debug.samTemplatePath": "The path to the SAM Template.",
+    "AWS.configuration.description.awssam.debug.samTemplateResource": "The LogicalID of the AWS::Lambda::Function or AWS::Serverless::Function to invoke.",
+    "AWS.configuration.description.awssam.debug.envvars": "Environment variables to pass to the function invocation (replaces template variables).",
+    "AWS.configuration.description.awssam.debug.lambda": "Lambda specific details of the invocation",
```

Comment:
Nit: use periods consistently. Some of these end in periods and others do not.

Also "Lambda-specific"

Reply:
Good point.